### DISCUSSION TITLE
How to add tolerance entities in ezdxf?

### DISCUSSION THREAD

**Discussion by luoyu-123** [mozman/ezdxf#318] [Q&A]
I want to use ezdxf to do a geometric tolerance function. When I use ezdxf to parse the dxf file exported from other software, I find that there is a tolerance entity attribute. How should I add tolerance entities in ezdxf?

**Comment by mozman** [comment]
https://ezdxf.mozman.at/docs/tutorials/linear_dimension.html#tolerances-and-limits
